### ISSUE TITLE
Verbose vs. truncated issue display

### ISSUE THREAD

**Issue by mekline** [psych-ds/psych-DS#44]
# TL;DR
Current verbose button doesn't do much, and medium-level information makes the drop-down-for-more functionality a little bit obscure

# Detailed View

* Verbose button should read: [ ] Verbose (expand all error messages and file lists)
* Clicking the Verbose button should result in all bubbles starting fully expanded.  
* Whether or not Verbose is clicked, user should still be able to click the triangles to expand/collapse individual errors/warnings

* Collapse view should show the error code and a truncated list of files that it applies to, e.g. ('|>' and '\/' is the triangle dropdown, light bars between Error, Reason, and Evidence should be maintained as is, ditto the boxes/spacing shown when expanded)

```
|> ERROR: CSV_COLUMN_MISSING
    /data/study-yarncolor_type-badnames_data.csv
    /data/subdir/subdir/study-yarn_location-subdir_data.csv
    /data/study-yarncolor_data.csv
    /data/whatever.csv
    /data/5thfile.csv      
    [...and 17 more files]

|> ERROR: CSV_FORMATTING_ERROR
    /data/study-validname_type-pdf_data.csv
```

* Expand view should show all information and all files, e.g. 

```
\/ ERROR: CSV_COLUMN_MISSING

Reason: A required column is missing

Evidence:

/data/study-yarncolor_type-badnames_data.csv

Column headers: [,garment,yarn_color] do not appear in variableMeasured. schema.rules.csv_data.Datafile

/data/study-yarncolor_data.csv

Column headers: [garment,yarn_color] do not appear in variableMeasured. schema.rules.csv_data.Datafile

/data/subdir/subdir/study-yarn_location-subdir_data.csv

Column headers: [yarn_color] do not appear in variableMeasured. schema.rules.csv_data.Datafile

...etc. 
```
